config.py

```diff
@@ -92,6 +92,80 @@ def load(self):
 The head of the HAProxy config. This contains global settings
 and defaults.
 '''))
+        self.add_template(
```

Comment:
I'm also not super concerned about the scenario you mentioned, because typically MLB will be run from inside a Docker container which only lives as long as the script is running. Any state inside the container is not preserved when it restarts.